terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/date/DateConvertUtils.java

```diff
@@ -83,7 +88,7 @@ public static Timestamp convertToTimestamp(java.util.Date date) {
      */
     public static Time convertToTime(java.util.Date date) {
         if (date == null) {
-            throw new IllegalArgumentException("date must not be null");
+            throw new IllegalArgumentException(MESSAGE_ARGUMENT_NULL_ERROR);
```

Comment:
Making only this message a constant only makes it less readable, and is essentially meaningless.
It's a good idea to undo the fix or standardize the argument checking.

Reply:
Created a standardized method.
1eb7f0f7b4f0982022e09afa25a8898e53d361f7